core/src/test/java/org/sonarsource/sonarlint/core/websocket/WebSocketServiceTest.java

```diff
@@ -59,9 +59,9 @@
 import static org.mockito.Mockito.verifyNoInteractions;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
-import static org.sonarsource.sonarlint.core.websocket.SonarCloudWebSocket.WEBSOCKET_DEV_URL;
 
 class WebSocketServiceTest {
+  private static final String WEBSOCKET_DEV_URL = "wss://events-api.sonarcloud.io/";
```

Comment:
this is prod url. is it intentional?

Reply:
This class will eventually disappear when tests are migrated to medium ones in this PR